Jenkinsfile

```diff
@@ -0,0 +1,19 @@
+node('lisk-explorer-01'){
+  stage ('Prepare Workspace') {
+    deleteDir()
+    checkout scm
+  }
+
+  stage ('Shellcheck') {
+    try {
+      sh '''#!/bin/bash -xe
+      # shellcheck
+      shopt -s globstar; /opt/shellcheck/shellcheck **/*.sh
+      '''
+      deleteDir()
+    } catch (err) {
```

Comment:
Looking at what this catch block does and given the whole `Jenkinsfile` is fairly simple we should probably use the [delcarative](https://jenkins.io/doc/book/pipeline/syntax/#declarative-pipeline) syntax.